app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileRepo.kt

```diff
@@ -37,6 +37,8 @@ class PinnedTileRepo(private val applicationContext: Application) {
             BehaviorSubject.create()
     val pinnedTiles: Observable<LinkedHashMap<String, PinnedTile>> = _pinnedTiles.hide()
 
+    val isEmpty: Observable<Boolean> = _pinnedTiles.map { it.size == 0 }.distinctUntilChanged()
```

Comment:
:+1:

Reply:
Could we add a line break before `distinctUntilChanged`, though?  It's Rx convention to put one operator on a line, although it's still pretty clear what's happening here.